### PR TITLE
added getID() to Credentials class

### PR DESCRIPTION
**Pull request by wolffles**
all done 👍 should I remove the GetID() from textDatasource now?

### CODE REVIEW

**Comment by wolffles** [comment]
I ran the prettier tests on my local system and it passed but here the CI says its failing. any idea what's going on?

File: package.json [hunk 1]
```diff
@@ -33,7 +33,8 @@
   },
   "homepage": "https://github.com/buttercup/credentials#readme",
   "dependencies": {
-    "@buttercup/signing": "^0.1.0"
+    "@buttercup/signing": "^0.1.0",
+    "hash.js": "1.1.7"
```
Comment (perry-mitchell) [inline package.json]: This version shouldn't be hardcoded.. it's fine to use the `^` prefix :)

File: test/unit/Credentials.spec.js [hunk 2]
```diff
@@ -157,4 +157,24 @@ describe("Credentials", function() {
...
+
+    describe("getID", function() {
+        it("should return a unique hash of this.data", function() {
+            const creds = new Credentials({
+                type: "text",
+                username: "bob",
+                password: "mypass123",
+                fake_value: "somevalue"
+            });
+            const creds2 = new Credentials({
```
Comment (perry-mitchell) [inline test/unit/Credentials.spec.js]: This isn't used currently..

**Comment by perry-mitchell** [comment]
Prettier is most likely failing because it's outdated, and the behaviour is different on different nodejs versions. You could try updating it to 1.16.1 in a new commit..

File: package.json (hunk 1 above)
Comment (wolffles) [reply package.json]: hah I always forget to change, when I use npm install

**Comment by wolffles** [comment]
updated. I might have forgot to run the format script.

**Comment by perry-mitchell** [comment]
Released in 1.2.0.

**Pull request merged**